tst/EnergyPlus/unit/ThermalComfort.unit.cc

```diff
@@ -998,3 +999,25 @@ TEST_F(EnergyPlusFixture, ThermalComfort_CalcThermalComfortPierceSET)
     EXPECT_NEAR(ThermalComfortData(1).PierceSET, 23.62, 0.01);
 
 }
+
+TEST_F(EnergyPlusFixture, ThermalComfort_CalcThermalComfortAdaptiveCEN15251)
+{
+    state.files.inputWeatherFileName.fileName = configured_source_directory() + "/tst/EnergyPlus/unit/Resources/ThermalComfortCEN15251Test.epw";
+
+    // test the initialisation
+    DayOfYear = 1;
+    CurrentYearIsLeapYear = false;
+    CalcThermalComfortAdaptiveCEN15251(state.files, true, true, 0.0);
+    EXPECT_NEAR(ThermalComfort::runningAverageCEN, -1.3671408, 0.01);
```

Comment:
Are these output values harvested from some other tool?  I assume they aren't just what the function happened to return, but the one below does have an awful lot of precision which makes me concerned.

Reply:
To be precise, these exact values are indeed from the ep simulation output, but I did validate them in both the excel manual calculation and a python script calculation directly from the epw file. However, rest assured, differences are only a matter of precision, and I suppose I can add a screenshot of the excel one for demonstration. One thing to note, the dry bulb temperature would be slightly different, as briefly explained in [the previous ASH55 PR](https://github.com/NREL/EnergyPlus/pull/6474#issue-168947418), leading to differences in these values that are small but uncertain.